packages/fxa-settings/config/webpack.config.js

```diff
@@ -664,6 +664,33 @@ module.exports = function (webpackEnv) {
           filename: 'static/css/[name].[contenthash:8].css',
           chunkFilename: 'static/css/[name].[contenthash:8].chunk.css',
         }),
+      isEnvProduction &&
```

Comment:
Should we do this all the time and not just prod?

Reply:
I could remove that but it wouldn't have an effect since that tag isn't injected and the css file isn't built for the dev env.